src/rp-schematron-base.sch

```diff
@@ -2179,6 +2179,10 @@
         <report test="ancestor::mixed-citation[@publication-type=('journal','data', 'patent', 'software', 'preprint', 'web', 'report', 'confproc', 'thesis', 'other')] and not(normalize-space(@person-group-type)=('','author'))" 
           role="warning" 
           id="ref-person-group-type-other">This <name/> inside a <value-of select="ancestor::mixed-citation/@publication-type"/> reference has the person-group-type '<value-of select="@person-group-type"/>'. Is that correct?</report>
+
```

Comment:
I think this `report` needs adding under a different `rule`. The current context is `ref//person-group`. Maybe it was meant to be added under 'ref-pub-id-checks'?